src/schemas/21-526EZ/schema.js

```diff
@@ -459,26 +340,11 @@ let schema = {
             maxLength: 100,
             pattern: "([a-zA-Z0-9\-'.#]([a-zA-Z0-9\-'.# ])?)+$"
           },
-          // Can we make this in to a dateRange?
-          startTreatmentDate: {
-            $ref: '#/definitions/datetime'
-          },
-          endTreatmentDate: {
-            $ref: '#/definitions/datetime'
+          treatmentDateRange: {
+            $ref: '#/definitions/dateRange'
           },
-          // Should this use a dropdown like address?
-          treatmentCenterCountry: {
-            type: 'string'
-          },
-          // Should this use a dropdown like address?
-          treatmentCenterState: {
-            type: 'string',
-            pattern: '[a-zA-Z]{2}'
-          },
-          treatmentCenterCity: {
-            type: 'string',
-            maxLength: 100,
-            pattern: "([a-zA-Z0-9\-'.#]([a-zA-Z0-9\-'.# ])?)+$"
+          treatmentCenterAddress: {
+            $ref: '#/definitions/treatmentCenterAddress'
           },
           treatmentCenterType: {
```

Comment:
It looks like there are two options here 'VA_MEDICAL_CENTER', 'DOD_MTF'. Does this only refer to treatments at VA centers? How should we format private center record request data?

Reply:
Oh yeah, look at that. :thinking: Does that mean `treatments` is only _meant_ to hold VAMCs? In which case, where do we put private medical records...

Great point, @cehsu.

Reply:
I believe it refers to both (i.e., both types of centers will use the same fields with the same schema / validations). That's the way it is in the Swagger, at least.

Reply:
Er, by "VAMCs," I meant VAMCs and DoD MTFs...
Are you saying this might also be used to hold private facilities as well?